registry/sql-registry/registry/database.py

```diff
@@ -53,8 +58,11 @@ def __init__(self, params):
     def make_connection(self):
         self.conn = pymssql.connect(**self.params)
 
-    def execute(self, sql: str, *args, **kwargs) -> list[dict]:
-        debug(f"SQL: `{sql}`")
+    def query(self, sql: str, *args, **kwargs) -> list[dict]:
+        """
+        Make SQL query and return result
+        """
+        warn(f"SQL: `{sql}`")
```

Comment:
>warn(f"SQL: `{sql}`") [](http://example.com/codeflow?start=8&length=21)

Not sure if it's intented to print every raw sql.

If it's intented, probably should log as verbose instead of warning

Reply:
Log level changed to debug